src/test/groovy/graphql/execution/incremental/IncrementalCallStateDeferTest.groovy

```diff
@@ -34,6 +34,7 @@ class IncrementalCallStateDeferTest extends Specification {
         results[2].incremental[0].data["a"] == "A"
     }
 
+    // flaky
```

Comment:
Is this trully flaky?

Reply:
I hade one failure during development but could not reproduce again so far. Will remove it